lib/messages.js

```diff
@@ -32,6 +35,20 @@ require('./config');
 // module permissions
 var PERMISSIONS = config.permission.permissions;
 
+config.scheduler.jobs.push({
+  id: 'bedrock-messages.jobs.cleanupJob',
+  type: 'bedrock-messages.jobs.cleanupJob',
+  schedule: config.messages.scheduler.frequency,
```

Comment:
Instead of "cleanupJob" let's rename this (and any other occurrences) to "batchMessages".